packages/teleterm/src/ui/services/quickInput/quickPickers.ts

```diff
@@ -130,43 +141,209 @@ export class QuickServerPicker extends ClusterPicker {
       serverUri: item.data.uri,
     });
   }
+
+  getAutocompleteResult() {
+    return null;
+  }
 }
 
-export class QuickCmdPicker implements QuickInputPicker {
-  launcher: CommandLauncher;
+export class QuickCommandPicker implements QuickInputPicker {
+  private pickerRegistry: Map<string, QuickInputPicker>;
 
-  constructor(launcher: CommandLauncher) {
+  constructor(private launcher: CommandLauncher) {
     this.launcher = launcher;
```

Comment:
I guess you forgot to remove these assignments 😄 (also in other places)

Reply:
Omg, sorry, I totally missed the fact that they're unnecessary. I thought it just gets rid of the class-level declarations but apparently I can also skip code in the constructor.